ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/integration/ClientIntegrationTest.java

```diff
@@ -541,7 +541,7 @@ public void shouldHandleErrorResponseFromExecuteQuery() {
     // Then
     assertThat(e.getCause(), instanceOf(KsqlClientException.class));
     assertThat(e.getCause().getMessage(), containsString("Received 400 response from server"));
-    assertThat(e.getCause().getMessage(), containsString("Missing WHERE clause"));
+    assertThat(e.getCause().getMessage(), containsString("A_FAKE_TABLE_NAME does not exist."));
```

Comment:
This test only cares that it gets an error. It was producing that error by running the query that we are now supporting. I replaced it with a query that will never work.